static/js/containers/CourseIndexPage.js

```diff
@@ -14,7 +15,7 @@ import {
   BannerImage
 } from "../components/PageBanner"
 import { Cell, Grid } from "../components/Grid"
-import SearchTextbox from "../components/SearchTextbox"
+import CourseSearchbox from "../components/CourseSearchbox"
```

Comment:
Not a big deal, wondering if maybe `SearchTextbox` should be renamed to something similar that reflects what's being searched, like `ChannelSearchBox`

Reply:
hm, we may want to re-name it, but it's still being used for the channel search and the site-wide search, so I think `SearchTextbox` is probably still accurate?

Reply:
OK